Readme.md

```diff
@@ -105,6 +105,9 @@ Files.write(Paths.get("TestReadme.result"), (result + "\n").getBytes());
 
 At the moment the following builtins are supported(and, by default, automatically injected when needed):
 
+- String
+    - `sprintf`
```

Comment:
Good! What I alluded to before is that it would be really good to mention here how this built-in won't behave exactly as `sprintf` in OPA, and should ideally only be used for simple operations, like `%s`, `%d` and such.

Reply:
Thanks for the feedback, I added a note, let me know how it reads :pray: